CMake/Platform/mbedOS-GNU-C.cmake

```diff
@@ -30,3 +30,9 @@ set(CMAKE_ASM_FLAGS_MINSIZEREL_INIT     "-Os -DNDEBUG")
 set(CMAKE_ASM_FLAGS_RELEASE_INIT        "-Os -DNDEBUG")
 set(CMAKE_ASM_FLAGS_RELWITHDEBINFO_INIT "-Os -g -gdwarf-3 -DNDEBUG")
 set(CMAKE_INCLUDE_SYSTEM_FLAG_ASM  "-isystem ")
+
+if(DEFINED YOTTA_CFG_DEBUG_OPTIONS_COVERAGE)
```

Comment:
I think it'd be better to check that this is both defined _and_ truthy, so that `"coverage": false` doesn't result in confusion. Just `if(YOTTA_CFG_DEBUG_OPTIONS_COVERAGE)` should do it.

From CMake docs:
> if(<constant>): True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number